apis/basic/composition.yaml

```diff
@@ -134,6 +141,8 @@ spec:
           patchSetName: network-id
         - type: PatchSet
           patchSetName: region
+        - type: PatchSet
+          patchSetName: network-id-tags
```

Comment:
nit: i would regroup and place this PatchSet next to `network-id`

Reply:
done